fission/install-windows-cli.ps1

```diff
@@ -0,0 +1,3 @@
+go install
```

Comment:
Can we remove this file?  You could test simply by having your gopath bin ahead of the rest of the path.